src/main/java/org/jclouds/vcloud/director/v1_5/config/VCloudDirectorHttpApiModule.java

```diff
@@ -57,4 +79,9 @@ protected void installLocations() {
       bind(ImplicitLocationSupplier.class).to(OnlyLocationOrFirstZone.class).in(Scopes.SINGLETON);
    }
 
+   @Override
+   protected void bindRetryHandlers() {
+      bind(HttpRetryHandler.class).annotatedWith(ClientError.class).to(VcloudDirectorClientErrorRetryHandler.class);
+      bind(HttpRetryHandler.class).annotatedWith(ServerError.class).to(VcloudDirectorServerErrorRetryHandler.class);
```

Comment:
not sure why you want to bind ServerError when VcloudDirectorServerErrorRetryHandler has retryableServerCodes always empy.

Possibly you can remote entirely line 85?